modules/ROOT/pages/authentication-authorization/load-privileges.adoc

```diff
@@ -79,4 +87,39 @@ SHOW ROLE roleLoadAllData PRIVILEGES AS COMMANDS
 a|Rows: 1
 |===
 
-The `LOAD ON ALL DATA` privilege is granted to the `PUBLIC` role by default.
\ No newline at end of file
+The `LOAD ON ALL DATA` privilege is granted to the `PUBLIC` role by default.
+
+[[access-control-load-cidr]]
+== The `CIDR` privilege
+
+_This feature was released in Neo4j 5.16._
+
+The load privilege on `CIDR cidr` enables or disables loading data from the given IPv4 or IPv6 CIDR range.
+If granted, the user can load data from sources in the given CIDR range.
+If missing or denied, no data can be loaded from sources in the given CIDR range.
+
+For example, the below query denies users with the role `roleLoadCidr` the ability to load data with `LOAD CSV` from `127.0.0.1/32`:
+
+[source, cypher, role=noplay]
+----
+DENY LOAD ON CIDR "127.0.0.1/32" TO roleLoadCidr
+----
+
+The below query lists all privileges for the role `roleLoadCidr` as commands:
+
+[source, cypher, role=noplay]
+----
+SHOW ROLE roleLoadCidr PRIVILEGES AS COMMANDS
+----
+
+.Result
+[options="header,footer", width="100%", cols="m"]
+|===
+|command
+|"DENY LOAD ON CIDR "127.0.0.1/32" TO `roleLoadCidr`"
+a|Rows: 1
+|===
+
+The `LOAD ON CIDR` privileges replace the `internal.dbms.cypher_ip_blocklist` configuration setting.
```

Comment:
Do we want to mention this config setting here? it is an internal one 🤔

Reply:
I don't think we should mention it, or is it mentioned somewhere else in the documentation?

Reply:
I did a quick search for it in the repo and didn't find anything

Reply:
Then I think it would be strange to mention it for the first time when we replace it.

Reply:
Yes, we don't document internal settings.